src/components/EnterpriseApp/data/hooks/useUpdateActiveEnterpriseForUser.js

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect, useCallback } from 'react';
+import { logError } from '@edx/frontend-platform/logging';
+
+import LmsApiService from '../../../../data/services/LmsApiService';
+
+const useUpdateActiveEnterpriseForUser = ({
+  enterpriseId,
+  user,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+
+  const updateActiveEnterpriseAndRefreshJWT = useCallback(async (username, currentEnterpriseId) => {
+    setIsLoading(true);
+    try {
+      await LmsApiService.getActiveLinkedEnterprise(username).then(async (linkedEnterpriseId) => {
+        if (linkedEnterpriseId !== currentEnterpriseId) {
+          await LmsApiService.updateUserActiveEnterprise(currentEnterpriseId);
+          await LmsApiService.loginRefresh();
```

Comment:
[curious] I'm wondering if we need the `loginRefresh` call here anymore now that we're no longer making assumptions of JWT role ordering. That is, I believe we were calling `loginRefresh` because our prior assumption that the JWT roles would be re-ordered.

Given that this hook no longer cares about the JWT roles, could we nix the call `login_refresh`?

Reply:
oh yeah, that's a great point! It's all done properly now 😄